modules/nw-networkpolicy-allow-external-clients.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-networkpolicy-allow-external-clients_{context}"]
+= Creating a multi-network policy to allow traffic from external clients
+
+With the `deny-by-default` policy in place you can proceed to configure a policy that allows traffic from external clients to a pod with the label `app=web`.
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role, then you can create a network policy in any namespace in the cluster.
+====
+
+Follow this procedure to configure a policy that allows external service from the public Internet directly or by using a Load Balancer to access the pod. Traffic is only allowed to a pod with the label `app=web`.
+
+.Prerequisites
+
+* Your cluster uses a cluster network provider that supports `NetworkPolicy` objects, such as
+ifndef::ovn[]
+the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+ifdef::ovn[]
+the OVN-Kubernetes network provider or the OpenShift SDN network provider with `mode: NetworkPolicy` set.
```

Comment:
This phrase is already present in older version but I'm not completely sure to understand it:

> Prerequisites
> - Your cluster uses a cluster network provider that supports NetworkPolicy objects, such as the OpenShift SDN network provider with mode: NetworkPolicy set. This mode is the default for OpenShift SDN.

@dougbtv @s1061123 Can you please confirm it is still a requirement for using multinetwork-policies?

(e.g. [4.11 docs multinetwork-policy](https://docs.openshift.com/container-platform/4.11/networking/multiple_networks/configuring-multi-network-policy.html))

Maybe we can fix it before duplicating, if needed

Reply:
@zeeke Yes I am building on previous version so I am not 100% sure if this statement is still relevant. The source code makes a distinction between ovn and sdn producing different output depending on the context.